omod/src/main/java/org/openmrs/module/owa/web/controller/OwaRestController.java

```diff
@@ -218,4 +218,12 @@
 		}
 		return appList;
 	}
+
+	@RequestMapping(value = "/rest/owa/allowAdmin", method = RequestMethod.GET)
+	@ResponseBody
+	public boolean allowWebAdmin() {
+		boolean allowwebadminprivileges;
```

Comment:
Please follow the OpenMRS [naming](https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions) conventions. In particular, the naming conventions on variables(hint: `camelCase`)

Reply:
on it